app/models/person.rb

```diff
@@ -103,6 +103,8 @@ class Person < ActiveRecord::Base
   has_many :authored_comments, :class_name => "Comment", :foreign_key => "author_id", :dependent => :destroy
   has_many :community_memberships, :dependent => :destroy
   has_many :communities, -> { where("community_memberships.status = 'accepted'") }, :through => :community_memberships
+  has_one  :community_membership, :dependent => :destroy
+  has_one  :community, -> { where("community_memberships.status= 'accepted'") }, through: :community_membership
```

Comment:
Or actually, some other, better, name that would convey the meaning of being the one where user is a member with citizenship.